lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -11,6 +13,9 @@
 import numpy as np
 
 
+image_comparison = partial(image_comparison, extensions=['png'])
```

Comment:
```suggestion
png_image_comparison = partial(image_comparison, extensions=['png'])
```
That caught me by surprise. I was scolling past this and just saw the decorators with removed `extensions` parameter. Please use `png_image_comparison` or `image_comparison_png`. Having the same function name but with different API is a bit unclear; in particular also because we `from ... import image_comparison` in other tests.

Reply:
sounds good, fixed